packages/shared/lib/contexts/onboarding/utils/canUserRecoverWithShimmerClaimingAccounts.ts

```diff
@@ -0,0 +1,11 @@
+import { IShimmerClaimingAccount, ShimmerClaimingAccountState } from '@contexts/onboarding'
+
+export function canUserRecoverWithShimmerClaimingAccounts(shimmerClaimingAccounts: IShimmerClaimingAccount[]): boolean {
```

Comment:
I would argue that `canUserRecoverFromClaiming` or `canUserRecover` are better and shorter alternatives while not losing any information on what the function does.

Reply:
Mmm to be consistent with everything else I left "Shimmer" in there, so it's in the middle: `canUserRecoverFromShimmerClaiming`